api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -1248,9 +1248,104 @@ const (
 // +kubebuilder:validation:Enum=Karpenter
 type Provisioner string
 
+// Configures when and how to scale down cluster nodes.
+type ScaleDownConfig struct {
+	// enabled determines whether the Cluster Autoscaler should scaled down this cluster.
+	//
+	// +kubebuilder:default=Enabled
+	// +kubebuilder:validation:Enum=Enabled;Disabled
+	// +optional
+	Enabled string `json:"enabled,omitempty"`
+
+	// delayAfterAddSeconds sets how long after scale up the scale down evaluation resumes in seconds
+	// It must be between 0 and 86400.
+	// When set to 0, this means scale down evaluation will resume immediately after scale up, without any delay.
+	// When omitted, this means no opinion, and the platform is left to choose a reasonable default.
```

Comment:
```suggestion
	// When omitted, this means no opinion, and the platform is left to choose a reasonable default, which is subject to change over time.
	// The current default is ...
```

Reply:
if omitted, the autoscaler defaults to 10 minutes.